packages/aws-cdk/bin/cdk.ts

```diff
@@ -69,6 +69,7 @@ async function parseCommandLineArguments() {
       .option('tags', { type: 'array', alias: 't', desc: 'Tags to add to the stack (KEY=VALUE)', nargs: 1, requiresArg: true })
       .option('execute', { type: 'boolean', desc: 'Whether to execute ChangeSet (--no-execute will NOT execute the ChangeSet)', default: true })
       .option('force', { alias: 'f', type: 'boolean', desc: 'Always deploy stack even if templates are identical', default: false })
+      .option('parameters', { type: 'array', alias: 'P', desc: 'Additional parameters passed to CloudFormation at deploy time (KEY=VALUE)', nargs: 1, requiresArg: true, default: {} })
```

Comment:
all our other options are aliased to lower case. Any reason this needs to be `P`?

Reply:
the lower `p` is used for plugins, which makes this crash. Unfortunately. :)

Reply:
I think it's better to leave aliases out for the time being and we can introduce it later if the need really arises in that case. thoughts?

Reply:
That could work. I only added it to fit in with the other options.